src/index.js

```diff
@@ -60,23 +64,28 @@ const createWithTranslation = (globalTranslations = {}, defaultLocale = DEFAULT_
             ...format
           }
 
-          const translateFunc = (key, translateOptions) => {
+          const translateFunc = (key, translateOptions = {}) => {
```

Comment:
I think you do not need a default, spreading should work just fine with `null`/`undefined`.